moban/jinja2/engine.py

```diff
@@ -136,3 +139,12 @@ def is_extension_list_valid(extensions):
         and isinstance(extensions, list)
         and len(extensions) > 0
     )
+
+
+def import_module_of_extension(extensions):
+    modules = set()
+    if extensions:
+        for extension in extensions:
+            modules.add(extension.split('.')[0])
```

Comment:
this line only helps jinjia_time but will not take action on 3rd_party.some_dir.import_me, because this logic only import 3rd_party but not import_me.

Please reference this method: https://github.com/chfw/lml/blob/dev/lml/utils.py#L54, (which is not public support by lml)

Reply:
For the present architecture, we only need to import the `3rd_party` part (since the user specifies the full extension name, _i.e._ `Jinja2_time.TimeExtension` instead of only `TimeExtension`).

Reply:
alright, let's go with current solution